content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -0,0 +1,121 @@
+---
+title: Menjalankan Multipel Instans dari Aplikasimu
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Penyekalaan aplikasi menggunakan kubectl.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+       <h3>Penyekalaan sebuah Aplikasi</h3>
+
+            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment yang dibuat itu hanya satu Pod yang menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu menyekalakan (<i>scale</i>) aplikasi kita untuk mengikuti tingkat permintaan pengguna.</p>
```

Comment:
nit! just to be clearer!
```suggestion
            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment yang dibuat hanya mempunyai satu Pod untuk menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu menyekalakan (<i>scale</i>) aplikasi kita untuk mengikuti tingkat permintaan pengguna.</p>
```

Reply:
```suggestion
            <p>Di modul-modul sebelumnya kita telah membuat <a href="/id/docs/concepts/workloads/controllers/deployment/">Deployment</a>, dan mengeksposnya secara publik via <a href="/id/docs/concepts/services-networking/service/">Service</a>. Deployment tersebut hanya membuat satu Pod untuk menjalankan aplikasi kita. Ketika kunjungan meningkat, kita perlu menyekalakan (<i>scale</i>) aplikasi kita untuk mengikuti tingkat permintaan pengguna.</p>
```